### DISCUSSION TITLE
0.6.1 missing features seen in screenshots ?

### DISCUSSION THREAD

**Discussion by julfur** [logchimp/logchimp#650] [General]
Hi @mittalyashu,

I'm interested in logchimp and tried to play with it without much knowledge in the technologies used (yarn, vuejs and such).

Quick feedback regarding the installation, the documentation can be perfected. I had to improvise at some points because the documentation is sometimes not coherent between steps.
Also, it is nor clear if docker is supported. I found a page describing the docker deployment but it's "hidden" and not listed in the installations available.

My main question was that it seems the last release (0.6.1) doesn't provide the features showed in the screenshot in the README.
I am currently missing the Search and Changelogs tabs on the homepage.
I also have issues with the default theme, the Roadmaps page width is very narrow and I have an unnecessary scrollbar and other page could take more screen width.

Also, as a site admin, I can't seem to be able to modify anything in the Settings > General page.
The save button is disabled.

And finally, I can't seem to find a way to assign a role I created to a specific user as a site admin.

Let me know If I should open issues regarding each point.
This tool needs a little more polish but it's already very usefull, thanks for your work.

Best regards.

**Comment by mittalyashu** [comment]
Hello @julfur,

Glad you found LogChimp to your liking 😄

> Quick feedback regarding the installation, the documentation can be perfected. I had to improvise at some points because the documentation is sometimes not coherent between steps.

What parts of the documentation do you think can be improved? Or the things you noticed which are not connecting together.

> Also, it is nor clear if docker is supported. I found a page describing the docker deployment but it's "hidden" and not listed in the installations available.

The LogChimp Docker image is still work in progress, it will mostly-likely be released along with LogChimp v0.7.x

> My main question was that it seems the last release (0.6.1) doesn't provide the features showed in the screenshot in the README.
I am currently missing the Search and Changelogs tabs on the homepage.

Yes, there are few features for which I've initially created design mock-ups but didn't got the chance to work on it.

> I also have issues with the default theme, the Roadmaps page width is very narrow and I have an unnecessary scrollbar and other page could take more screen width.

At what screen viewport you were felt the width of roadmap was narrow?

> Also, as a site admin, I can't seem to be able to modify anything in the Settings > General page.
The save button is disabled.

I can't recall if this is an expected behaviour or a bug, I've to look into it. _(it's been long time since I touched that part of the codebase)_

> And finally, I can't seem to find a way to assign a role I created to a specific user as a site admin.

I would say that's the expected behaviour for now in UI, since I'm still evaluating this feature.
